### DISCUSSION TITLE
Possibility to set up relationships between previously unmanaged database tables

### DISCUSSION THREAD

**Discussion by leonceaklin** [directus/directus#14701] [Feature Requests]
To make Directus 9 work with existing databases or databases that were managed with Directus 8, it should be easy to redefine the structure of an existing database in the data model editor. As of now, it's possible to set field types but not relationships. I suggest implementing a feature which makes it possible to set the type of an unmanaged field to a m2o or o2m relationship field as well as defining an unmanaged table as m2m junction table so users don't have to manually edit the directus_relations and directus_fields table.

**Comment by rijkvanzanten** [comment]
> so users don't have to manually edit the directus_relations and directus_fields table.

Not a solution, but just a FYI, you could use the `/relations` endpoint to manually create/remove relationships currently. Slightly easier than manually editing the database 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
